src/easylink/graph_components.py

```diff
@@ -45,7 +45,13 @@ class InputSlot:
     """A function that validates the input data being passed into the pipeline via 
     this ``InputSlot``. If the data is invalid, the function should raise an exception 
     with a descriptive error message which will then be reported to the user.
-    **Note that the function must be defined in the** :mod:`easylink.utilities.validation_utils` 
+    **Note that the function *must* be defined in the** :mod:`easylink.utilities.validation_utils` 
+    **module!**"""
+    splitter: Callable[[list[str], str, Any], None] | None = None
+    """A function that aggregates and then splits the incoming data to this ``InputSlot``
```

Comment:
```suggestion
    """A function that splits the incoming data to this ``InputSlot``
```

I understand what you mean by this, but I think it is more a property of the specific splitter you've written, and I don't think omitting it is incorrect.